README.windows.md

```diff
@@ -20,7 +20,7 @@ Alternatively, you may prefer the features of a more full-function IDE, such as
 
 # Binary distribution
 
-Julia runs on Windows XP-SP2 and later (including Windows Vista, Windows 7, and Windows 8).
+Julia runs on Windows 7, 8 and 10.
```

Comment:
Does Julia no longer work on XP? Also, it might be nice to use the Oxford comma here for consistency with the sentence as it previously was, i.e. a comma after 8.